TrkHitReco/src/FlagBkgHits_module.cc

```diff
@@ -72,9 +77,11 @@ namespace mu2e
       float                                       cperr2_;
       int const                                   debug_;
       std::string                                 kerasW_;
+      bool                                        useSLine_;
       float                                       kerasQ_;
       int                                         iev_;
-      std::shared_ptr<TMVA_SOFIE_TrainBkgDiag::Session> sofiePtr;
+      std::shared_ptr<TMVA_SOFIE_TrainBkgDiag::Session> sofiePtr1;
```

Comment:
Where does this model definition come from?  I see the #include for the Chi2SLine model, but not for the other model.   Should this module #include all the models?

Reply:
Line #21 and #22 have the definitions for the models. Chi2 model has additional parameters.